config/locales/en/views/hearings.yml

```diff
@@ -19,7 +19,7 @@ en:
           no_hearing_details: No hearing details available
       hearing_type: Hearing type
       pagination:
-        next_page: Next hearing day
-        previous_page: Previous hearing day
+        next_page: Next
+        previous_page: Previous
```

Comment:
Did dan request this change? He had explicitly requested "Next/Previous hearing day" before

Reply:
I have just checked with Dan and he agreed with this change as you would now be able to page through the days in different (non-chronological) order.